### ISSUE TITLE
Спасибо большое за перевод!

### ISSUE THREAD

**Issue by Wizardzim** [AgentMC/Sao18#2]
**AgentMC** видел, сегодня вы завершили свой перевод 18 тома. 
Спасибо большое! Прочитал ваше послесловие переводчика. Очень тяжело делать несвойственную себе работу, особенно в первый раз. За это отдельное спасибо. Я если бы и попробовал такое явно бы не выдержал и забросил на полпути, а у вас получилось, и это очень здорово. 
К сожалению, у меня туго с английским, и я не мог вам помочь с переводом.
Пробовал переводить каждый кусочек, выкладываемый на английском с помощью гугл транслейта на русский, но получалось очень плохо.
Но своё утро я начинал с чтения нового кусочка вашего перевода, который вы выкладывали как правило по ночам (для Украины) прямо из diff-ов в коммитах.
Я не привередлив к переводу, и очень рад что ваш перевод вышел гораздо раньше, чем мог быть бы любой другой перевод.
В любом случае, чтобы не писали потом набежавшие критики, которым то одно, то другое режет слух, неправильные обороты речи, мне перевод очень понравился. Я понял о чём хотел написать автор, и это для меня главное. Если где-то и были какие-то огрехи, меня они никак не задели.
Ещё раз огромная, сердечная благодарность. То что вы делали было не зря.
P.S. Знаю, отписавшихся будет в сотни раз меньше, чем читателей, а большинство отписавшихся ещё и будет недовольны, но знайте - так всегда получается. На самом деле читатели довольны. Таков уж род человеческий.

**Comment by Prizvel** [comment]
Присоединяюсь к благодарности. 
Пару вопросов по редактуре:
1. Планируется ли оформить прямую речь по правилам русского языка или так и останется в кавычках?
2. Может заменить некоторые термины на те, что использовал Ушвуд. В частности "Рыцари целостности" заменить на "Рыцари Единства",  ИИ "внизу-вверх"/"сверху-вниз" заменить на восходящий/нисходящий ИИ.

**Comment by AgentMC** [comment]
@Wizardzim спасибо за тёплые слова! А как же всё форматирование? Я так старался... :)

@Prizvel, я думаю, что за исключением каких-то жутких ошибок, которые я пропустил, больше ничего в целом меняться не будет. Но поконкретнее: 

1. Нет, по 3 причинам:
    * Это колоссальный объём работы, не несущей дополнительной полезной нагрузки, кроме пунктуации конкретного языка.
    * Тире в этом тексте — важный элемент, описывающий "подвисшую паузу". 
    * Дык а что прикажете делать с удалённой речью (двойные квадратные кавычки)? Не хотелось бы терять такой инструмент передачи речи.
2. Тут можно каждый индивидуально рассмотреть. В целом, мне просто привычнее Аркнарок. 
    * У Арка есть запись, где он подровно описывает почему тот или иной термин переводится так или иначе. Задумайтесь, к чему стремится Администратор, создавшая Рыцарей. Как носитель программы Кардинала, она движема необходимостью обеспечить целостность системы (а не единство людей). И рыцари поэтому — целостности, а не "единства", "всецелого" или ещё как.
    * Похоже, это официальный термин. Принято. **Создайте, пожалуйста, отдельую issue для разбора терминологии** с вашими предложениями, там и обсудим.

**Comment by Wizardzim** [comment]
@AgentMC _А как же всё форматирование? Я так старался... :)_
Абзацы и пунктуацию видно и так, а в целом - это же Markdown, его и так воспринимать несложно

**Comment by Prizvel** [comment]
@AgentMC, Хорошо, так и поступлю. Так же возможно отправлю пул реквест по поводу ещё нескольких моментов, не связанных с терминами, если конечно разберусь как это делается :))
P.S. Сегодня дочитал, ещё под впечатлением. Теперь можно строить новые теории по Ускоренному Миру))

**Comment by danbestman** [comment]
Это не переводчик,  тоже читатель....

24 августа 2017, 23:34:56, от Wizardzim :

**Comment by danbestman** [comment]
Я не переводчик,  это мейл читателя 

25 августа 2017, 11:18:17, от Prizvel :

**Comment by ThanksFor18Volume** [comment]
AgentMC, доброго времени суток)
Зарегистрировался, чтобы сказать спасибо за проявленную Вами инициативу и проделанную работу. Перевод вышел хороший, насладился чтением. Пусть даже 75% от дословности.. это не так важно, по сравнению со светлыми эмоциями, оставшимися после прочтения финала истории от Кавахары-сэнсэя)

**Comment by klavomaster** [comment]
Хочу сказать огромное человеческое спасибо за перевод. Обычно никогда ничего не комментирую, но тут не могу промолчать ^^

**Comment by eugeneko** [comment]
@AgentMC Пресвятые огуречики, ты прекрасен!
После того, как Ашвуд слился, я почти потерял надежду дочитать этот чертов SAO.
Пролистывал defan-а и taptrans-а в надежде увидеть хоть что-то новое, прочитал последнюю пару томов в их переводе, но они идут очень далеко от онгоинга.
И тут я встречаю этот репозиторий.

**Comment by JenikabBlack** [comment]
Зарегистрировалась, чтобы поблагодарить за перевод 18 тома. 
Спасибо Вам за огромную проделанную работу! Так обидно было, что перевода именно последней книги не было ни на русском, ни на английском, а так хотелось узнать, чем закончилась арка. 
От лица тех, кто не умер от томительного любопытства - огромное человеческое спасибо!)
П.с. Вы планируете переводить следующие тома, 19-20?

**Comment by AgentMC** [comment]
@JenikabBlack https://github.com/AgentMC/Sao19 ;-)

**Comment by eugeneko** [comment]
@AgentMC 
/*счастливые вопли*/

**Comment by Yorgyo** [comment]
Да, всё выше сказанное верно, я тоже прочитал 18 том только благодаря вам. Спасибо большое) Присоединяюсь к читателям 19 тома, в вашем переводе)

**Comment by zloywolk** [comment]
Огромное спасибо за перевод. Вы провели поистенне колосальную работу. Буду ждать следующие ваши переводы. С уважением и с наилучшими пожеланиями.

**Comment by GoldaAlexey** [comment]
Читаю 18 том в Вашем переводе, спасибо большое)

**Comment by Okiys** [comment]
Столько времени прошло а ваш "неточный" перевод до сих пор актуален)) спасибо за работу.

После послесловия хочу сказать что некоторые необъясненности которые вы указывали объяснялись до этого или же не требуют объяснения,  но даже так соглашусь, что были необъясненные события, но это или упущения автора, или же я не увидел ответ на них.
А еще благодаря вам (до этого не задумывался) увидел как рыцарей по настоящему назвал автор в оригинале, что тоже немного раскрыло мир)))

**Comment by fabul97** [comment]
Большое спасибо, было очень вкусно. 🍡
